pkg/apis/config/types.go

```diff
@@ -90,3 +101,14 @@ type TargetLoadPackingArgs struct {
 	// Address of load watcher service
 	WatcherAddress string
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NodeResourceTopologyMatchArgs holds arguments used to configure the NodeResourceTopologyMatch plugin
+type NodeResourceTopologyMatchArgs struct {
+	metav1.TypeMeta
+
+	KubeConfigPath string
+	MasterOverride string
+	Namespace      string
```

Comment:
Hmm... Why a plugin config is designed to be restricted to one namespace?

Reply:
Maybe the limitation of only one namespace is not so flexible. I'll provide a list of namespaces this plugin can work with.